### PR TITLE
sdcicd-1200 Add a blocking job for osde2e

### PR DESCRIPTION
**Pull request by YiqinZhang**
- Create a blocking job which will run the new step registry with osde2e provision, test, and cleanup steps.

- The blocking suite use CONFIGS=rosa,sts,stage,blocking-suite.

Card : [SDCICD-1200](https://issues.redhat.com/browse/SDCICD-1200)

### CODE REVIEW

**Comment by YiqinZhang** [comment]
/pj-rehearse periodic-ci-openshift-osde2e-main-nightly-4.15-rosa-classic-sts-blocking

**Comment by YiqinZhang** [comment]
/pj-rehearse periodic-ci-openshift-osde2e-main-nightly-4.15-rosa-classic-sts-blocking

File: ci-operator/jobs/openshift/osde2e/openshift-osde2e-main-periodics.yaml [hunk 1]
```diff
@@ -1758,6 +1758,69 @@ periodics:
...
+  extra_refs:
+  - base_ref: main
+    org: openshift
+    repo: osde2e
+  labels:
+    ci-operator.openshift.io/variant: nightly-4.15
+    ci.openshift.io/generator: prowgen
+    job-release: "4.15"
+    pj-rehearse.openshift.io/can-be-rehearsed: "true"
+  name: periodic-ci-openshift-osde2e-main-nightly-4.15-rosa-classic-sts-blocking
```
Comment (mmazur) [inline ci-operator/jobs/openshift/osde2e/openshift-osde2e-main-periodics.yaml:1776]:
```suggestion
  name: periodic-ci-openshift-osde2e-main-nightly-4.15-blocking-rosa-classic-sts
```

**Comment by YiqinZhang** [comment]
/pj-rehearse periodic-ci-openshift-osde2e-main-nightly-4.15-conformance-rosa-classic-sts

**Comment by YiqinZhang** [comment]
/pj-rehearse periodic-ci-openshift-osde2e-main-nightly-4.15-conformance-rosa-classic-sts

**Comment by YiqinZhang** [comment]
/pj-rehearse periodic-ci-openshift-osde2e-main-nightly-4.15-conformance-rosa-classic-sts

**Comment by YiqinZhang** [comment]
/pj-rehearse periodic-ci-openshift-osde2e-main-nightly-4.15-conformance-rosa-classic-sts

**Comment by YiqinZhang** [comment]
/pj-rehearse periodic-ci-openshift-osde2e-main-nightly-4.15-conformance-rosa-classic-sts

**Comment by YiqinZhang** [comment]
/pj-rehearse periodic-ci-openshift-osde2e-main-nightly-4.15-conformance-rosa-classic-sts

**Pull request merged**